core/server/themes/loader.js

```diff
@@ -0,0 +1,39 @@
+var debug = require('debug')('ghost:themes:loader'),
+    config = require('../config'),
+    events = require('../events'),
+    read = require('./read'),
+    settingsApi = require('../api/settings'),
+    updateConfigAndCache,
+    loadThemes,
+    initThemes;
+
+updateConfigAndCache = function updateConfigAndCache(themes) {
+    debug('loading themes', themes);
+    config.set('paths:availableThemes', themes);
```

Comment:
I assume the idea is, that we are getting rid of adding the themes to the config **soon**.
But first we need to merge this PR, then add the api endpoint, then adapt the client and finally remove the whole logic of remembering and updating themes in the settings cache.

Am i right/wrong?Would like to hear your thinking to this :)